### ISSUE TITLE
Installation of v0.7.4.1 fails on clean win 7

### ISSUE THREAD

**Issue by MarcinOrlowski** [CipherShed/CipherShedBuilds#4]
I tried install v0.7.4.1 on clean Win 7 box and installer failed. Installation was as-is, no option disabled in settings. First, the dialog "Unable to find required procedure" shows up (it looks like system dialog as it is transladed), and then when I hit "OK" it shows another dialog:

> Failed to install 'C:\Windows\system32\Drivers\truecrypt.sys`. Do you want to continue installing?

**Comment by GigabyteProductions** [comment]
This is a regression that's being worked on now.

**Comment by MarcinOrlowski** [comment]
Where can I find older builds?

**Comment by GigabyteProductions** [comment]
Same place; other branches. I think the last one before 0.7.4.1 is [https://github.com/CipherShed/CipherShedBuilds/tree/v0.7.4.0-20151231](https://github.com/CipherShed/CipherShedBuilds/tree/v0.7.4.0-20151231).

**Comment by MarcinOrlowski** [comment]
Thanks. 0.7.4.0 installs fine.

**Comment by UIfric** [comment]
Will ask here... What about Windows 10? Does CipherShed support that new UEFI , I mean - can it encrypt whole system boot partitions with  boot authentication?

**Comment by GigabyteProductions** [comment]
EFI support is a work in progress with a new installer.

**Comment by jult** [comment]
I don't think whole system disk encryption is the way to go, to be honest. Volumes (as huge files) that are encrypted are a much better idea, and they can function as RAMdisk at the same time, which can be made advantageous. You should have your system set up so that an opened (decrypted) volume is auto-dismounted (in encrypted state only) when you consider access to your system by untrusted parties a valid danger, be that in hybernation, sleep mode, shutdown etc.

**Comment by GigabyteProductions** [comment]
While file containers can be easier, there are a lot of nearly out of control ways that information can leak out of them when the running environment is storing data unencrypted. Some examples include indexing, Windows registry, Windows explorer history, ever storing the data unencrypted (even temporarily), swap files and especially hibernation. Encrypting the entire disk is a security must.

**Comment by xaminmo** [comment]
FYI, if you unpack 7.4.1 to the install dir, and then install, even with the errors, it puts truecrypt.sys where it needs to be, and seems to work fine.

**Comment by xaminmo** [comment]
Regarding "Encrypting the entire disk is a security must" really should read "encrypting the entire system..."   Because I could encrypt whole drives, but not the system drive, and still run into the same issues pointed out by Kyle.

**Comment by GigabyteProductions** [comment]
You are correct. I didn't consider multi-drive configurations at the time of writing.
